packages/core/src/request.ts

```diff
@@ -45,6 +45,25 @@ export function sessionToSentryRequest(session: Session, api: API): SentryReques
   };
 }
 
+/** Creates a SentryRequest from an Aggregates of Request mode sessions */
+export function sessionAggregatesToSentryRequest(sessionAggregates: SessionAggregates, api: API): SentryRequest {
+  const sdkInfo = getSdkMetadataForEnvelopeHeader(api);
+  const envelopeHeaders = JSON.stringify({
+    sent_at: new Date().toISOString(),
+    ...(sdkInfo && { sdk: sdkInfo }),
+  });
+  // The server expects type `sessions` in headers for Session Aggregates payload
+  const itemHeaders = JSON.stringify({
+    type: 'sessions',
+  });
+
+  return {
+    body: `${envelopeHeaders}\n${itemHeaders}\n${JSON.stringify(sessionAggregates)}`,
+    type: 'sessions' as SentryRequestType,
```

Comment:
`SetryRequest` should be modified to include `sessions` as one of the type.

Reply:
I did not want to do that because If I do that it will mess with https://github.com/getsentry/sentry-javascript/blob/master/packages/node/src/transports/base.ts#L47 because it will be expected to add it here, and I am not rate limiting SessionAggregates

Reply:
You can change it there to `Omit<SentryRequestType, 'sessions'>`